includes/free-trial/partners/class-wc-calypso-bridge-partner-square.php

```diff
@@ -0,0 +1,124 @@
+ <?php
+
+/**
+ * WC Calypso Bridge Partner Square
+ *
+ *	@since   2.3.4
+ *	@version 2.3.4
+
+ * This file includes customizations for the sites that were created through /start/square on woo.com.
+ * woocommerce_onboarding_profile.partner must get 'square'
+ */
+class WC_Calypso_Bridge_Partner_Square {
+
+	/**
+	 * Class instance.
+	 *
+	 * @var WC_Calypso_Bridge_Partner_Square instance
+	 */
+	protected static $instance = false;
+
+	/**
+	 * Get class instance
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		// Only for free trials.
+		if ( ! wc_calypso_bridge_is_ecommerce_trial_plan() ) {
+			return;
+		}
+		$onboarding_profile = get_option( 'woocommerce_onboarding_profile', array() );
+		if ( ! isset( $onboarding_profile['partner'] ) ) {
+			return;
+		}
+
+		if ( $onboarding_profile['partner'] !== 'square' ) {
+			return;
+		}
+
+		$this->add_square_setup_task();
+		$this->add_square_connect_url_to_js();
+		$this->remove_woo_payments_from_payments_suggestions_feed();
+	}
+
+	/**
+	 * Remove woo payments from the payments suggestions feed.
+
```

Comment:
Sorry, extra space. Follow-up nit
```suggestion
	 *
```